src/ia-account-settings.interface.ts

```diff
@@ -0,0 +1,81 @@
+export interface IAAccountSettingsInterface {
+  /**
+   * set screenname in UserModel
+   *
+   * @param {Event} e
+   * @memberof IAAccountSettingsInterface
+   */
+  setScreenname(e: Event): void;
+
+  /**
+   * set email in UserModel
+   *
+   * @param {Event} e
```

Comment:
q: are these just regular events or custom event? if custom, can we talk about its `e.detail` value/shape?

Reply:
these are regular events.